media-sound/teamspeak-server/files/teamspeak.service-r1

```diff
@@ -9,8 +9,8 @@ Group=teamspeak
 WorkingDirectory=/opt/teamspeak3-server
 Environment="LD_LIBRARY_PATH=/opt/teamspeak3-server"
 Environment="TS3SERVER_LICENSE=accept"
-ExecStartPre=/bin/rm -f /dev/shm/7gbhujb54g8z9hu43jre8
-ExecStart=/opt/bin/ts3server inifile=/etc/teamspeak3-server/ts3server.ini
+ExecStartPre=rm -f /dev/shm/7gbhujb54g8z9hu43jre8
+ExecStart=ts3server inifile=/etc/teamspeak3-server/ts3server.ini
```

Comment:
I didn't know systemd started using PATH lookup. Does it really work for /opt/bin, though?

Reply:
Ouch, I wasn't aware of this. Reverted back.